task/02-numbers-tasks.js

```diff
@@ -127,7 +134,7 @@ function getAngleBetweenVectors(x1, y1, x2, y2) {
  *     0     => 0
  */
 function getLastDigit(value) {
-    throw new Error('Not implemented');
+    return `${value}`.split('').pop();
```

Comment:
do it with remainder operator - %

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder